src/new/connection.ts

```diff
@@ -66,6 +67,41 @@ class WalletConnection extends BaseConnection {
     this.setState(ConnectionState.CONNECTING);
     this.websocket.setup();
   }
+
+  startControlHandlers(storage: IStorage) {
```

Comment:
The parent class uses a generic `options: any` so that subclasses can use the parameters they want.
For instance, maybe the wallet service connection would like to add some parameters to control how the connection start, like the auth token to authenticate when the connection starts.